### DISCUSSION TITLE
Example with existing vitejs

### DISCUSSION THREAD

**Discussion by mirrorps** [Advaith3600/seamless-admin#5] [General]
Great package, thanks! 

Suggestion: will be great if there is an example to use the package with an existing vitejs setup. Currently cannot find an easy way to do this. One can override the default layout, but then there're vitejs compile errors, so an example will help. The easy way is to touch the SeamlessAdminServiceProvider::bootDirectives() but this is a total antipattern.

**Comment by Advaith3600** [comment]
I will be sure to provide some examples. Currently, completely changing the layout is not supported out of the box, but many other configurations are there. Anyone reading this is welcome to share some examples .
